src/NzbDrone.Core/Localization/Core/en.json

```
@@ -1017,6 +1017,8 @@
   "IndexerSettingsRejectBlocklistedTorrentHashesHelpText": "If a torrent is blocked by hash it may not properly be rejected during RSS/Search for some indexers, enabling this will allow it to be rejected after the torrent is grabbed, but before it is sent to the client.",
   "IndexerSettingsRssUrl": "RSS URL",
   "IndexerSettingsRssUrlHelpText": "Enter to URL to an {indexer} compatible RSS feed",
+  "IndexerSettingsSeasonPackSeedRatio": "Season Pack Seed Ratio",
+  "IndexerSettingsSeasonPackSeedRatioHelpText": "The ratio a season-pack torrent should reach before stopping, empty uses the download client's default. Ratio should be at least 1.0 and follow the indexers rules",
   "IndexerSettingsSeasonPackSeedTime": "Season-Pack Seed Time",
   "IndexerSettingsSeasonPackSeedTimeHelpText": "The time a season-pack torrent should be seeded before stopping, empty uses the download client's default",
```

```suggestion
  "IndexerSettingsSeasonPackSeedRatio": "Season Pack Seed Ratio",
  "IndexerSettingsSeasonPackSeedRatioHelpText": "The ratio a season pack torrent should reach before stopping, empty uses the download client's default. Ratio should be at least 1.0 and follow the indexers rules",
  "IndexerSettingsSeasonPackSeedTime": "Season Pack Seed Time",
  "IndexerSettingsSeasonPackSeedTimeHelpText": "The time a season pack torrent should be seeded before stopping, empty uses the download client's default",
```
for the sake of consistency 😄

---

thanks, nice catch